src/command.rs

```diff
@@ -60,22 +60,20 @@ impl SubstrateCli for Cli {
         Ok(match id {
             "dev" => Box::new(chain_spec::general::develop_config()),
             "local" => Box::new(chain_spec::general::local_config()),
-            "alcyone-dev" => Box::new(chain_spec::alcyone_testnet::develop_config()),
-            "alcyone-local" => Box::new(chain_spec::alcyone_testnet::local_config()),
+            "testnet-dev" => Box::new(chain_spec::testnet::develop_config()),
```

Comment:
let's revert these. The user should always have to enter the specific testnet (Alcyone/ITN) that they have to join.

Reply:
Can you please comment on how to revert this? `testnet-dev` is a particular config of the specific testnet called `testnet`.  Perhaps the name doesn't look very distinctive and we could rename `testnet` to something less neutral?

Reply:
This can be left as it is, I wanted `"Testnet" | "testnet" | ""` to be reverted since that config points to Alcyone, not the generic testnet config.

Reply:
`"testnet"` points to a modified Alcyone config for which I created two fresh JSON files (plain and raw). So strictly speaking `testnet` is not Alcyone. There is still Alcyone as a JSON spec, even though I've replaced it in `chain_spec.rs` with `testnet`.

Reply:
Let's get rid of the `testnet` jsons and only have `alcyone` jsons.